site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -65,54 +79,312 @@ LAST SEEN   TYPE      REASON              OBJECT
 5s          Warning   ResourceDeleted     nginxgateway/ngf-config                          NginxGateway configuration was deleted; using defaults
 ```
 
+##### Get shell access to NGINX container
+
+Getting shell access to containers allows developers and operators to view the environment of a running container, see its logs or diagnose any problems. To get shell access to the NGINX container, use `kubectl exec`:
+
+```shell
+kubectl exec -it -n nginx-gateway  <ngf-pod-name> -c nginx /bin/sh
+```
+
 ##### Logs
 
 Logs from the NGINX Gateway Fabric control plane and data plane can contain information that isn't available to status or events. These can include errors in processing or passing traffic.
 
-To see logs for the control plane container:
+1. To see logs for the:
```

Comment:
This numbering looks off to me in the deploy preview:
<img width="1053" alt="Screenshot 2024-06-27 at 4 36 40 PM" src="https://github.com/nginxinc/nginx-gateway-fabric/assets/50597707/6929d81b-4a63-48ff-8abb-198cc2cd8b1a">

Reply:
I have removed the usage of numbers to sub-headings. Will wait on hearing from Alan.

Reply:
seems fixed now, thanks @kate-osborn